README.md

```diff
@@ -464,6 +470,7 @@ Contributions are always welcome!
 * [SRECon EMEA 2019 Recap](https://speakerdeck.com/dastergon/srecon-emea-2019-recap-sre-muc-meetup)
 * [Life of an SRE at Google - JC van Winkel](https://www.youtube.com/watch?v=7Oe8mYPBZmw)
 * [Site Reliability Engineering for Native Mobile Apps - Abhijith Krishnappa](https://www.infoq.com/articles/site-reliability-engineering-mobile-apps/) - Case study: Halodoc adaptation of SRE principles for Native Mobile Apps
+* [SRE Best Practices](https://www.infracloud.io/blogs/sre-best-practices/)
```

Comment:
It would nice add from which team it mean best practices. 
For example:  `SRE Best Practices by Infracloud` or `SRE Best Practices by Google`. It would remove confusion and misunderstandings.

Reply:
@anjuls can you address the recommendation?

Reply:
I have updated as per the recommendation by @miry. Thanks.